From e1e2c31bca42d78f2e83a910241ef126c44c437a Mon Sep 17 00:00:00 2001
From: Jiffin Tony Thottan <jthottan@gmail.com>
Date: Mon, 28 Mar 2016 16:39:24 +0530
Subject: [PATCH 35/80] build : fixing dependency issue for glusterfs-ganesha in RHEL6

glusterfs-ganesha requires "dbus-send" binary, but location of this
binary is different in RHEL6 and RHEL7. For RHEL6 it is /bin/ and
RHEL7 it is /usr/bin/. So the patch will add dependency based on
package dbus-1 not on the binary dbus-send

Upstream reference
>Change-Id: I266af7b983791be648749f01a04b38479d6f7c03
>BUG: 1322242
>Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>
>Reviewed-on: http://review.gluster.org/13848
>Smoke: Gluster Build System <jenkins@build.gluster.com>
>Reviewed-by: Niels de Vos <ndevos@redhat.com>
>NetBSD-regression: NetBSD Build System <jenkins@build.gluster.org>
>CentOS-regression: Gluster Build System <jenkins@build.gluster.com>
>cherry-picked as 71d988919c75e1294eb417a92f0a495cf388002e

Change-Id: I266af7b983791be648749f01a04b38479d6f7c03
BUG: 1321544
Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/70854
Reviewed-by: Niels de Vos <ndevos@redhat.com>
Tested-by: Niels de Vos <ndevos@redhat.com>
---
 glusterfs.spec.in |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index 7e9d7f3..7366bcd 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -394,8 +394,7 @@ Summary:          NFS-Ganesha configuration
 Group:            Applications/File
 
 Requires:         %{name}-server%{?_isa} = %{version}-%{release}
-Requires:         nfs-ganesha-gluster
-Requires:         pcs, /usr/bin/dbus-send
+Requires:         nfs-ganesha-gluster, pcs, dbus
 
 %description ganesha
 GlusterFS is a distributed file-system capable of scaling to several
-- 
1.7.1

